### PR TITLE
[ITS] Speed-up CA initialisation + 2 iteration tracking example

### PR DESCRIPTION
**Pull request by mpuccio**
I prepared this PR with @sgorbuno who kindly offered help in speeding up some of the routines in the tracking. This time the initialisation was optimised as well as the performance macro (CheckTracks.C) and our own steering macro.
In the new version of the steering macro there is also an example of the 2(n) iterations tracking. I still have to port the full param set of the second iteration from AliRoot, it will come in a second PR.

### CODE REVIEW

**Comment by davidrohr** [comment]
Ciao @mpuccio : we have added the fast atan2 to O2 common mathutils. Could you invoke it from there instead of using your own copy? It should also work on GPU already (I hope)

**Comment by mpuccio** [comment]
Ciao @davidrohr, fixed!

File: Detectors/ITSMFT/ITS/tracking/include/ITStracking/Cluster.h [hunk 1]
```diff
@@ -29,17 +29,19 @@ namespace its
 {
 
 struct Cluster final {
+  Cluster() {} // = default;
```
Comment (davidrohr) [inline Detectors/ITSMFT/ITS/tracking/include/ITStracking/Cluster.h]: I think when you comment out the C++11 initialization values below, you should comment out the whole constructor (or make it = default).
In any case, the code checker warnings should be fixed.

**Comment by mpuccio** [comment]
I did not see it, the output of the codechecker is completely unreadable, is there a better way of displaying it? I will fix that.

**Comment by davidrohr** [comment]
@mpuccio : What I do is check the details log and search for "List of errors found", that is the relevant part...

**Comment by mpuccio** [comment]
Ah, at the top of the log file... why does it go on if it already finds errors?
It should be fixed now...

**Comment by mpuccio** [comment]
I force pushed with a squashed version that preserve a ~sensible history. Let me know if it is fine with you ;)

**Comment by davidrohr** [comment]
Thx
Sure, just wanted to get rid of the fixup commits. Will merge once CI is green.

**Comment by davidrohr** [comment]
@ktf @MichaelLettrich : Could you check what is the issue with travis for this PR? It is stuck since >4 hours.

**Comment by ktf** [comment]
The check actually completed, but it looks like travis is having troubles to communicate back to Github nowadays... :-/ @awegrzyn did you experience the same?

**Pull request merged**